### DISCUSSION TITLE
rdtclient update not working

### DISCUSSION THREAD

**Discussion by ketzerkadaver** [tteck/Proxmox#1855] [Help Needed] [answered]
Hello friends,

The update script fails: ✗ No RDTClient Installation Found!

The update for rdtclient won't work, because the script is looking for  /opt/rdtclient/ which is actually /opt/rdtc/.

Thank you for all those awesome scripts!

**Comment by tteck** [comment]
fixed

**Comment by ketzerkadaver** [comment]
Awesome! Blazingly fast! Thank you!
Edit:
But after executing the script it still shows: Version v2.0.39 of RealDebrid Client was found. You are currently on version v2.0.37.
Do I need to pay attention to anything else?

**Comment by ketzerkadaver** [comment]
Found the solution:
The framework needs to be updated first. 6 ->8

I managed to get the update working by:
```
service rdtc stop
wget -q https://github.com/rogerfar/rdt-client/releases/latest/download/RealDebridClient.zip
unzip -qq RealDebridClient.zip -d /opt/rdtc
rm RealDebridClient.zip
cd /opt/rdtc
mkdir -p data/db/ data/downloads
sed -i 's#/data/db/#/opt/rdtc&#g' /opt/rdtc/appsettings.json
service rdtc start
```

Could you implemet that?


Thanks a lot for your awsome work!

**Reply by tteck** [reply, marked as answer]
fixed

The ASP.NET Core Runtime updated to 8 using the Debian 12 .deb (`https://packages.microsoft.com/config/debian/12/packages-microsoft-prod.deb`)

created update path
```
systemctl stop rdtc
mkdir -p rdtc-backup
cp -R /opt/rdtc/appsettings.json rdtc-backup/
wget -q https://github.com/rogerfar/rdt-client/releases/latest/download/RealDebridClient.zip
unzip -oqq RealDebridClient.zip -d /opt/rdtc
cp -R rdtc-backup/appsettings.json /opt/rdtc/
rm -rf rdtc-backup RealDebridClient.zip
systemctl start rdtc
```
_Should_ be able to type `update` in the LXC console to update moving forward

For the ones that need to update ASP.NET Core Runtime to 8, `apt-get install -y dotnet-sdk-8.0`
